Cargo.toml

```diff
@@ -25,6 +25,7 @@ warp = { version = "*", default-features = false }
 
 [dev-dependencies]
 tempfile = "*"
+pax = "*"
```

Comment:
yeah, I don't think you want to do this. I think you just want to `cargo install pax` in the build scripts.

Reply:
Might need to add the installed binary to PATH if it doesn't work automatically too.